src/Schema/Factories/NodeFactory.php

```diff
@@ -4,22 +4,24 @@
 
 use GraphQL\Type\Definition\Type;
 use GraphQL\Type\Definition\EnumType;
+use GraphQL\Type\Definition\UnionType;
```

Comment:
By length

Reply:
@spawnia  oh.. I just saw many PHP files in this project actually are sorted by length, and some of them sorted by vendor, if this is the rule I'll follow 👍